### PR TITLE
Preparation for release 0.1.0-pre-alpha-08

### LINKED ISSUE #63: Refactoring component registry
To have components we are in need of with LVN

### PR DESCRIPTION
**Pull request by nelson-glauber**
Updates before releasing a new version on [JitPack](https://jitpack.io/)

### CODE REVIEW

**Review by simlay — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# LiveView Native Core Jetpack
+
+This library provides an abstraction layer on top of the [LiveView Native Core](https://github.com/liveview-native/liveview-native-core) library.
+
+## Pre-requisites
+
+In order to build this library, it's necessary to following steps:
```
Comment (supernintendo) [inline README.md]: Just a small typo here:

```suggestion
In order to build this library, it's necessary to do the following steps:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Use the latest version of [Android Studio](https://developer.android.com/studio) with [NDK](https://developer.android.com/studio/projects/install-ndk).
+- This project contains RUST files which depending on **LiveView Core libray** and exposes functionalities to the Kotlin layer via [JNI](https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/jniTOC.html). Therefore, you need to [install RUST](https://www.rust-lang.org/tools/install).
```
Comment (supernintendo) [inline README.md]: Some more typos:

```suggestion
- This project contains Rust files which depends on **LiveView Core library** and exposes functionality to the Kotlin layer via [JNI](https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/jniTOC.html). Therefore, you need to [install Rust](https://www.rust-lang.org/tools/install).
```

**Review by simlay — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- After install RUST, you'll need to install the toolchains for each platform which the library will be generated *(arm, arm64, x86, x86_64, darwin-x86-64, darwin-aarch64)*. This project is using [RUST Gradle Plugin](https://github.com/mozilla/rust-android-gradle), therefore follow the steps described in the correspondent section in their website. For instance:
```
Comment (supernintendo) [inline README.md]:
```suggestion
- After installing Rust, you'll need to install the toolchains for each platform which the library will be generated *(arm, arm64, x86, x86_64, darwin-x86-64, darwin-aarch64)*. This project is using [Rust Gradle Plugin](https://github.com/mozilla/rust-android-gradle), therefore follow the steps described in the corresponding section in their website. For instance:
```

**Review by supernintendo — COMMENTED** [review]
Just some typos in the README, but other than that looks good to me!

**Pull request merged**